README.md

```diff
@@ -11,29 +11,73 @@ theory behind these constructors is researched in
 [DOT](http://www.cs.uwm.edu/~boyland/fool2012/papers/fool2012_submission_3.pdf),
 a calculus for dependent object types.
 
-The dotty compiler is largely a new design. It takes a more functional
-approach than current scalac, paired with aggressive caching to
-achieve good performance. Some parts, in particular those that have to
-do with configuration and input/output are ported from the Scala
-compiler. The compiler is fully functional, in particular it can
-compile itself. But there's more work needed (and planned) on
-polishing rough edges, improving diagnostics, improving compilation
-speed, and embedding in other tools.
-
-We expect that, over time, some of the new technologies explored in
-this project will find their way into future versions of Scala. At
-present it is too early to say which ones and when.
-
-If you want to try it out, to get started have a look at
+####Current status:
+ _Technology preview_: currently unsupported, may not be functionally complete, and are not suitable for deployment in production.
+
+####Is it going to be the future Scala?
+Yes, eventually.
+
+####Who's working on it?
+See [github contributors page](https://github.com/lampepfl/dotty/graphs/contributors).
+ 
+####What are the features that could make me consider trying it?  
+| Feature                                         	| Status              	|
+|-------------------------------------------------	|---------------------	|
+| Union, Intersection and Literal singleton types 	| Implemented         	|
+| Fast compilation (phase fusion)                 	| Implemented         	|
+| Working contravariant implicits                 	| Implemented         	|
```

Comment:
I'd say In progress since https://github.com/lampepfl/dotty/pull/1246 has not been merged yet and the rules are still somewhat weird as you said in https://github.com/lampepfl/dotty/issues/1247#issuecomment-217713122